v3/src/components/graph/adornments/lsrl/lsrl-adornment-component.tsx

```diff
@@ -168,14 +166,14 @@ export const LSRLAdornment = observer(function LSRLAdornment(props: IProps) {
           // If there are multiple lines and equations, we offset the equations slightly
           .style("top", `${screenY + linesIndex * 20}px`)
       } else {
-        const left = equationCoords.x * 100
-        const top = equationCoords.y * 100
-        equation.style("left", `${left}%`)
-          .style("top", `${top}%`)
+        const left = equationCoords.x
+        const top = equationCoords.y
+        equation.style("left", `${left}px`)
+          .style("top", `${top}px`)
       }
     }
-  }, [cellKey, dataConfig, equationContainerSelector, getLines, model, plotHeight, plotWidth, showConfidenceBands,
-      xAttrId, xAttrName, xScale, xSubAxesCount, yAttrId, yAttrName, yScale, ySubAxesCount])
+  }, [cellKey, dataConfig, equationContainerSelector, getLines, interceptLocked, model, plotHeight, plotWidth,
+      showConfidenceBands, xAttrId, xAttrName, xScale, xSubAxesCount, yAttrId, yAttrName, yScale, ySubAxesCount])
```

Comment:
I do think we need to invalidate this when the specified dependencies change since changes to them have an effect on how the adornment's elements are rendered. Similar to my other comment specifically about `interceptLocked`, I expect it'd be possible to change that by making some significant changes to the component as whole and how it renders things, but I'm not sure and tend to think I should move on for now. If you think this is a critical area that needs improvement, though, I could probably spend some more time investigating that.